rrfs-test/testinput_expr/rrfs_mpasjedi_2024052700_letkf.yaml

```diff
@@ -85,9 +85,3 @@ output mean prior:
 output: # for outputting mean posterior
   filename: ./ana.$Y-$M-$D_$h.$m.$s.nc
   stream name: analysis 
-
```

Comment:
the same question to those changes,, why online verifications were removed?

Reply:
See my response above.